pkg/app/app.go

```diff
@@ -25,10 +26,16 @@ var (
 	// routed.
 	NotFound UI = &notFound{}
 
-	routes = make(map[string]UI)
-	uiChan = make(chan func(), 256)
+	routes   = make(map[string]UI)
+	routesRe = make([]regexRoute, 0)
```

Comment:
I would like to avoid shortcut names for regular expressions. `Re` does not express enough the meaning of what it does.

`routesWithRegexp` would work.

Reply:
good suggestion. will do

Reply:
good suggestion. will do